docs/CHANGELOG.asciidoc

```diff
@@ -780,6 +781,9 @@ APM::
 * Adds pagination to source map API {kibana-pull}145959[#145959]
 * Adds ability to offset point labels on maps {kibana-pull}145773[#145773]
 
+Cases::
+* Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]
```

Comment:
Could you move the following into the Cases section from the Alerting section:

- Redesign all cases list select modal [#149851](https://github.com/elastic/kibana/pull/149851)
- Users can now click a button on Case Detail and All Cases List to copy a case’s UUID to the clipboard [#148962](https://github.com/elastic/kibana/pull/148962)
- Sorting, severity filter and status filter are now persisted in the URL and local storage for the all cases list [#148549](https://github.com/elastic/kibana/pull/148549)
- Sorting by status, severity, and title is now allowed in the all-cases list [#148193](https://github.com/elastic/kibana/pull/148193)
- See "My recently assigned cases" to the recent cases widget of Security Solution dashboard [#147763](https://github.com/elastic/kibana/pull/147763)
- Adds the ability to bulk edit assignees on multiple cases [#146907](https://github.com/elastic/kibana/pull/146907)
Save draft user comment [#146327](https://github.com/elastic/kibana/pull/146327)

Reply:
Addressed in [902920f](https://github.com/elastic/kibana/pull/157805/commits/902920f25e01eb0671b455f72fc4b4cfab7fddc3).